README.md

```diff
@@ -5,52 +5,59 @@
 7. Run `npm run build` in the project's root. This will create a `dist` folder that contains all the files needed to run this demo. 
-8. Add the host `jwplayer.vuplay.local.drm.technology` to your local machine's hosts file.
-9. Load a supported browser and go to `https://jwplayer.vuplay.local.drm.technology:14703`
+8. Add the host `jwplayer.studiodrm.local` to your local machine's hosts file.
+9. Load a supported browser and go to `https://jwplayer.studiodrm.local:14703`
 
-You will need an enterprise license with JW Player in order to playback DRM encrypted content. Please contact support@vualto.com or [JW Player](https://www.jwplayer.com/pricing/) for more information. 
+You will need an enterprise license with JW Player in order to playback DRM encrypted content. Please contact <support@jwplayer.com> or see [JW Player pricing](https://www.jwplayer.com/pricing/) for more information. 
 
-To retrieve the JW Player key [click here](https://dashboard.jwplayer.com/#/players/downloads) and copy the license key for the appropriate player in the `Downloads` section.
+Retrieve the [JW Player key](https://dashboard.jwplayer.com/#/players/downloads) and copy the license key for the appropriate player in the `Downloads` section.
 
 In order to allow DRM encrypted playback in chrome (<https://goo.gl/EEhZqT>), SSL has been enabled for the demo. You will get a warning about an invalid cert `NET::ERR_CERT_AUTHORITY_INVALID` but this can safely be ignored.
 
 ### Browser support
 
 The browser must support [encrypted media extensions](https://www.w3.org/TR/2016/CR-encrypted-media-20160705/).
 Currently this includes the latest versions of Chrome, Firefox, Internet Explorer 11 and Edge.
-For a complete breakdown of supported media extensions please contact support@vualto.com
+For a complete breakdown of supported media extensions please contact <support@jwplayer.com>
 
 ## Useful links
 
-### VUDRM
+### Studio DRM
 
--   [Contact vualto](https://www.vualto.com/contact-us/)
--   [VUDRM](https://vudrm.vualto.com/)
--   [VUDRM token documentation](https://docs.vualto.com/projects/vudrm/en/latest/VUDRM-token.html)
+-   [Contact JW Player](https://support.jwplayer.com/)
+-   [Studio DRM](https://developer.jwplayer.com/jwplayer/docs/studio-drm-standalone-getting-started)
+-   [Studio DRM token documentation](https://developer.jwplayer.com/jwplayer/docs/studio-drm-token-v2)
+
+### JW Player
+
+-   [Homepage](https://www.jwplayer.com/)
+-   [Pricing](https://www.jwplayer.com/pricing/)
+-   [Developer Portal](https://developer.jwplayer.com)
+-   [JW Player documentation on the configuration of DRM with JWPlayer](https://developer.jwplayer.com/jw-player/docs/developer-guide/customization/configuration-reference/#drm)

```

Comment:
should this link be this one: https://developer.jwplayer.com/jwplayer/docs/studio-drm-standalone-web-players
as this is for standalone?

Reply:
or even here: https://developer.jwplayer.com/jwplayer/docs/studio-drm-standalone-web-players#jw-player

Reply:
Actioned.